.github/workflows/bazel-cache-push.yml

```diff
@@ -92,9 +100,8 @@ jobs:
           run: |
             cd /workspaces/magma
             bazel build //...
-            bazel test //...
-            bazel test //... --config=asan
-            bazel test //... --config=lsan
+            bazel test //... --cache_test_results=no
```

Comment:
just curious: is there a reason why the run here executes all statements vs for the vm where all statements are split in different steps?

Reply:
good q! just inconsistent. Will fix!